devolearn/cell_membrane_segmentor/cell_membrane_segmentor.py

```diff
@@ -138,6 +138,8 @@ def predict(self, image_path, pred_size = (350,250), centroid_mode = False):
         im = cv2.imread(image_path,0)
 
         tensor = self.preprocess(im)
+
+        self.model.segmentation_head[1].recompute_scale_factor = None
```

Comment:
Can you leave in some comments above this line to explain what exactly is the reason behind this? This is to help make it less confusing for others. (same goes for the other diff)

Reply:
done